src/main.ts

```diff
@@ -30,21 +30,19 @@ const resize = (): void => {
     const maxWidth = 1920;
     const maxHeight = 1080;
 
-    const width = Math.min(window.innerWidth, maxWidth);
-    const height = Math.min(window.innerHeight, maxHeight);
-
-    canvas.width = width;
-    canvas.height = height;
+    // Set the canvas dimensions to 1920x1080
+    canvas.width = maxWidth;
```

Comment:
Would this make the canvas be drawn in FullHD even if the window is actually smaller? Could this make it slower on smaller screens?